src/averageAges.js

```diff
@@ -54,8 +84,25 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
-function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+function calculateAverageAgeDiff(people, onlyWithSon = 0) {
+  let allChildren = [...people].filter(
```

Comment:
method filter returns a new array
```suggestion
  let allChildren = people.filter(
```